### PR TITLE
fix: change icon to symbol

### PR DESCRIPTION
**Pull request by ealvesss**
See [JENKINS-69715](https://issues.jenkins.io/browse/JENKINS-69715).

### Before
Screen Shot 2022-10-06 at 19 40 46

### After
File addtion:
Screenshot 2022-10-17 at 22 18 21

File modification:
Screenshot 2022-10-17 at 22 18 41

File deletion:
Screenshot 2022-10-17 at 22 19 02

### Proposed changelog entries

- Replace 'Changes' view icon with a symbol.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7229"></a>

### CODE REVIEW

**Review by NotMyFault — COMMENTED** [review]
Hey,

thanks for your PR! The icon has three states, according to the corresponding Java class: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/scm/EditType.java#L57-L59

The appropriate icon is filled in depending on the SCM operation that took place. We should keep that up, rather than replacing the icons with a generic symbol, no?

**Comment by EltonAlvess** [comment]
> eplacing the icons with a generic symbol, no

What do you suggest? Long time I dont contribute with jenkins. 

I'm trying to learn about the structure again.

**Comment by NotMyFault** [comment]
I went ahead and cleaned up the improper merge. Although, I'm not too sure about changing the reference from 'delete' to 'trash', because it's available in Jelly.

**Comment by EltonAlvess** [comment]
> I went ahead and cleaned up the improper merge. Although, I'm not too sure about changing the reference from 'delete' to 'trash', because it's available in Jelly.

Any change needed?

File: core/src/main/java/hudson/scm/EditType.java
Comment (daniel-beck) [inline core/src/main/java/hudson/scm/EditType.java:59]: This looks like a breaking change for the remote access API.

**Review by daniel-beck — DISMISSED** [review]
As above.

File: core/src/main/java/hudson/scm/EditType.java
Comment (EltonAlvess) [reply core/src/main/java/hudson/scm/EditType.java]: I've changed to delete again. Any other issue?

**Review by NotMyFault — APPROVED** [review]
I went ahead and updated the screenshots matching the new output, looks good to me, thanks for the PR.

**Review by uhafner — APPROVED** [review]
It is better than before! I'm not sure if it is a good idea to reuse the pencil, as this icon has a meaning for me: to actively edit something. But since there is no +/- symbol available right now I am fine with this change (in the Git forensics plugin I am using this one: https://github.com/jenkinsci/forensics-api-plugin/blob/master/src/main/resources/images/symbols/diff-stat.svg).

**Comment by NotMyFault** [comment]
Would someone from @jenkinsci/core-security-review mind taking a look, please? :)

**Review by Kevin-CB — APPROVED** [review]
Sorry for the delay, from a security aspect, I see no issues with this PR 🙂

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
